app/controllers/users_controller.rb

```diff
@@ -145,16 +145,13 @@ def mark_as_invited(user)
   end
 
   def create_institution_invite
-    invited_user_email = params[:user_invite_data]["email"]
-    unless invited_user_email.nil?
-      pending_invite = PendingInstitutionInvite.new
-      pending_invite.invited_user_email = invited_user_email
-      pending_invite.invited_by_user = current_user
-      pending_invite.institution_name = params["institution_data"]["name"]
-      pending_invite.institution_kind = params["institution_data"]["type"]
-      pending_invite.save!
-      @pending_invite = pending_invite
-    end
+    pending_invite = PendingInstitutionInvite.new
+    pending_invite.invited_user_email = params[:user_invite_data]["email"]
+    pending_invite.invited_by_user = current_user
+    pending_invite.institution_name = params["institution_data"]["name"]
+    pending_invite.institution_kind = params["institution_data"]["type"]
+    pending_invite.save!
+    @pending_invite = pending_invite
```

Comment:
The guard for `invited_user_email.nil?` is unnecessary. If that value is nil, it blows up before reaching this method. So I figure this simplification would be nice.